code/modules/surgery/organs/tongue.dm

```diff
@@ -322,8 +324,8 @@
 	name = "cat tongue"
 	desc = "A rough tongue, full of small, boney spines all over it's surface."
 	say_mod = "meows"
-	disliked_food = VEGETABLES | SUGAR | CLOTH
-	liked_food = DAIRY | MEAT
+	disliked_food = GROSS | VEGETABLES | SUGAR | CLOTH
+	liked_food = DAIRY | MEAT | GORE
```

Comment:
I don't think having felinids like organs etc really makes sense. They're still human, they just can't eat chocolate

Reply:
Sure. I wanted to do this in the first place, but was wary of making too radical a change.

Want me to move RAW to their disliked_food as well? 
like: `disliked_food = GROSS | RAW | VEGETABLES | SUGAR | CLOTH

Never sat right with me that felinids could consume raw, parasite-infested meat the same as a lavalander, despite them being genetically like 98% human and having no evolutionary disposition for needing to do so.

(im partial to removing the dislike for vegetables too. If they arent strict carnivores, why does it matter)

Reply:
> I don't think having felinids like organs etc really makes sense. They're still human, they just can't eat chocolate

Cats are carnivores and require meat based diets to survive... I dont see the harm in letting them eat organs and raw meat as its taken directly from animaistic traits... also kinda funny picturing it.
They also cant eat Veg or sugar as a trade off so....?
I really dont see why this is an issue Kat... just feels like a spite removal from a species while the dm file is open.

<details><summary>Observe</summary>

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/75050485/a486e792-0706-4b3c-84ab-1a44a72087c7)


</details>

Reply:
⠀🥺
👉👈

M-muh organ diet... You HAVE to reconsider!!!!! 😭😭😭

Reply:
If it's really that big a deal let's leave it how it is with this PR. No rotting stuff but organs are ok